depmapomics/config.py

```diff
@@ -190,9 +190,9 @@
                'MEAN_LOG2_COPY_RATIO': 'Segment_Mean',
                'CALL': 'Status'}
 
-SOURCE_RENAME = {'CCLF': 'Broad WES', 'CHORDOMA': 'Chordoma WES',
-                'SANGER': 'Sanger WES', 'IBM': 'Broad WES',
-                np.nan: 'Broad WES', 'DEPMAP': 'Broad WES',
+SOURCE_RENAME = {'CCLF': 'Broad', 'CHORDOMA': 'Chordoma',
```

Comment:
is this going to remove the difference between CCLE and Broad sources? or will the old CCLE lines still be marked as CCLE?

Reply:
this is a debug. it won't change anything to the way things were. we were just listing some WGS as CCLE WES instead of CCLE WGS..